app.js

```diff
@@ -9,6 +9,20 @@ const app = express();
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
+
+app.use((req, res, next) => {
```

Comment:
Hi @vincentayorinde Nice work. But why did you add line 13 to 24

Reply:
Fixed. Thank you